README.md

```diff
@@ -1,3 +1,42 @@
 # medic-builds-repo
 
 Design doc for builds repository.
+
+DDOCS are [pushed up by travis](https://github.com/medic/medic-webapp/blob/master/scripts/ci/push_to_staging.sh) with a specific name. These are then [pulled back down by api](https://github.com/medic/medic-api/blob/master/controllers/upgrade.js) and deployed by [horticulturalist](https://github.com/medic/horticulturalist).
+
+## Naming scheme
+
+Three valid examples: `medic:medic:3.0.0-beta.1`, `medic:medic:3.0.0`, `medic:medic:my-awesome-feature-branch`.
+
+The first two chunks between `:` is the namespace and the application name respectively. The namespace is just that: it has no logic relevant and is just used to scope things. The application name maps to the original ddoc name. It is what will be used to determine what to call the ddoc when deploying it.
```

Comment:
I don't follow this point. Firstly it's entirely appropriate to hardcode the ddoc name. If we decide to split things up or change the name we will hardcode the new name.

Secondly there's nothing here that talks about the db name so there's no coupling between ddoc and db names.

I think using the term ddoc name is a little confusing - this is the primary ddoc (which I've been calling the application). It bundles the api and sentinel modules, as well as our medic-client and medic-conflicts ddocs. Nothing is stopping us from breaking out more ddocs in the future and bundling them together into one file for deployment.

This piece of documentation is just talking about each app in the builds db having a unique ID and a version. The  unique ID for our main app is `medic:medic`. We could also ship `medic:reporter` and `medic:couch2pg` and anything else we want at a later date.